components/navigation/header.js

```diff
@@ -86,7 +86,7 @@ const Header = () => {
             </a>
             <a
               className="btn fw-500 btn-primary rounded-pill"
-              href="https://share.hsforms.com/1fstvMCeZRZKTYA4nG1VTPgcq0j9"
+              href="https://www.getcollate.io/"
```

Comment:
Can we take this link from constant, so in future we don't have to make changes in multiple files?

Reply:
Sure

Reply:
Done